core/cap-0046-02.md

```diff
@@ -124,72 +120,34 @@ The maximum WASM contract size will be introduced as a `ConfigSettingEntry`(see
 
 It is set during the protocol version upgrade using a new `ConfigSettingEntry`,
 with `configSettingID` == `CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES`, and
-`contractMaxSizeBytes` == `16384`. The valid values for
-`contractMaxSizeBytes` are[0, 256000] (inclusive).
-
-#### No contract-driven WASM installation
-
-This CAP does not specify a way to install WASM sources from within a
-contract. 
-
-This is done to encourage efficient code reuse and deduplication: if the
-contract was allowed to install the WASM code, then we'd need to store it
-twice (in the installer contract and in the source entry).
+`contractMaxSizeBytes` == `65536`. The valid values for
```

Comment:
I think this we'll start with smaller limits for the contract size and max entry size, but probably that can be updated later.